packages/design-tokens/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Just clarifying the reason this file exists is to get the right types in local development when dealing with files in this directory?

It seems that the options here don't overlap with the dist version, can we combine them?

Reply:
Yep that's correct - to get the same config in both dev and dist.

The dist extends this one, but it has extra config in it specifically only required for dist and not for dev usage, thus keeping the separation.